HueBoo/RootViewController.swift

```diff
@@ -71,13 +71,13 @@ class RootViewController: UIViewController, RootPresentable, RootCollectionViewL
     private func setupViews() {
         
         collectionView.listener = self
-        collectionView.backgroundColor = .red
+        collectionView.backgroundColor = .red // when I swipe back on the first cell I can see this. Might be better to use black as a default?
         
         view.addSubview(collectionView)
         
         NSLayoutConstraint.activate([
-            collectionView.topAnchor.constraint(equalTo: view.topAnchor),
-            collectionView.bottomAnchor.constraint(equalTo: view.bottomAnchor),
+            collectionView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
```

Comment:
Seems to me like the safeAreaLayoutGuide would be most applicable to the top and bottom anchors because of the notch, but maybe I'm missing something